docs/conf.py

```diff
@@ -132,6 +140,7 @@ def setup(app: Sphinx) -> None:
     "python": ("https://docs.python.org/3", None),
     "discord": ("https://discordpy.readthedocs.io/en/master/", None),
     "aiohttp": ("https://docs.aiohttp.org/en/stable/", None),
+    "statsd": ("https://statsd.readthedocs.io/en/v3.3/", ("_static/statsd_additional_objects.inv", None)),
```

Comment:
The custom objects.inv file is needed here since statsd's objects.inv file only references the aliased statsd.StatsClientBase, and does not have a reference to the "real" object at statsd.base.StatsClientBase, so intersphinx complains.